pkgs/development/python-modules/yamlloader/default.nix

```diff
@@ -16,6 +16,7 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [
+    hypothesis
     pytest
```

Comment:
That could be switched to pytestCheckHook.

Reply:
I don't have much will to fix it right now :(